redirects.yaml

```diff
@@ -16,6 +16,213 @@ docs/snap/2.7/cli/how-to-manage-networks: /docs/snap/2.7/cli/network-discovery
 docs/snap/2.7/ui/how-to-manage-networks: /docs/snap/2.7/ui/network-discovery
 docs/snap/2.8/cli/how-to-manage-networks: /docs/snap/2.8/cli/network-discovery
 docs/snap/2.8/ui/how-to-manage-networks: /docs/snap/2.8/ui/network-discovery
+docs/disk-erasure: /docs/snap/3.1/ui/storage
+docs/explore-maas: /docs/snap/3.1/ui/maas-installation
+docs/get-started-with-maas: /docs/snap/3.1/ui/give-me-an-example-of-maas
+docs/getting-help: /docs/snap/3.1/ui/getting-help
+docs/hardware-test-scripts: /docs/snap/3.1/ui/hardware-test-scripts-reference
+docs/hardware-testing: /docs/snap/3.1/ui/how-to-test-machines
+docs/high-availability: /docs/snap/3.1/ui/high-availability
+docs/how-to-manage-networks: /docs/snap/3.1/ui/how-to-manage-networks
+docs/images: /docs/snap/3.1/ui/images
+docs/install-from-a-snap: /docs/snap/3.1/ui/maas-installation
+docs/install-from-packages: /docs/deb/3.1/ui/installation
+docs/install-with-lxd: /docs/snap/3.1/ui/how-to-manage-vm-hosts#heading--lxd-setup
+docs/installation: /docs/snap/3.1/ui/maas-installation
+docs/installconfig-block: /docs/snap/3.1/ui/storage
+docs/installconfig-checklist: /docs/snap/3.1/ui/maas-installation
+docs/installconfig-images-import: /docs/snap/3.1/ui/select-and-import-images
+docs/installconfig-images-mirror: /docs/snap/3.1/ui/local-image-mirror
+docs/installconfig-images-vmware: /docs/snap/3.1/ui/vmware-images
+docs/installconfig-images: /docs/snap/3.1/ui/images
+docs/installconfig-install: /docs/snap/3.1/ui/maas-installation
+docs/installconfig-iso-install: /docs/install-from-iso
+docs/installconfig-lxd-install: /docs/install-with-lxd
+docs/installconfig-network-dev-discovery: /docs/snap/3.1/ui/about-networking#heading--about-network-discovery
+docs/installconfig-network-dhcp: /docs/snap/3.1/ui/how-to-manage-dhcp
+docs/installconfig-network-ipranges: /docs/snap/3.1/ui/ip-ranges
+docs/installconfig-network-ipv6: /docs/ipv6: /docs/snap/3.1/ui/about-networking#heading--about-ipv6
```

Comment:
Something went wrong here and on the next few lines you've got multiple left-hand-sides